Makefile

```diff
@@ -13,7 +13,7 @@ LD=gcc
 # Compile flags for linux / osx
 ifeq ($(uname_S),Linux)
 	SHOBJ_CFLAGS ?=  -fno-common -g -ggdb
-	SHOBJ_LDFLAGS ?= -shared -Bsymbolic -Bsymbolic-functions
+	SHOBJ_LDFLAGS ?= -shared -Wl,-Bsymbolic,-Bsymbolic-functions
```

Comment:
I think -Bsymbolic-functions is redundant, but if you wanna keep it you need to introduce it with it's own -Wl.